examples/dtls-echo/Makefile

```diff
@@ -24,51 +24,54 @@ BOARD_INSUFFICIENT_MEMORY := airfy-beacon b-l072z-lrwan1 bluepill calliope-mini
 # NOTE: 6LoWPAN will be included if IEEE802.15.4 devices are present
 USEMODULE += gnrc_netdev_default
 USEMODULE += auto_init_gnrc_netif
-# Specify the mandatory networking modules for IPv6 and sUDP
+# Specify the mandatory networking modules for IPv6 and UDP
 USEMODULE += gnrc_ipv6_router_default
-USEMODULE += gnrc_udp
-# Add a routing protocol
+USEMODULE += gnrc_sock_udp
+# Add a routing protocol (optional)
 USEMODULE += gnrc_rpl
-# This application dumps received packets to STDIO using the pktdump module
-USEMODULE += gnrc_pktdump
+
 # Additional networking modules that can be dropped if not needed
 USEMODULE += gnrc_icmpv6_echo
 # Add also the shell, some shell commands
 USEMODULE += shell
 USEMODULE += shell_commands
 USEMODULE += ps
 
-#TinyDTLs (crypto.c) made use of pthread
- ifneq ($(BOARD),native)
-   USEMODULE += pthread
- endif
+USEPKG += tinydtls
 
-# Comment this out to disable code in RIOT that does safety checking
-# which is not needed in a production environment but helps in the
-# development process:
-DEVELHELP ?= 1
+# Uncomment to redefine port. Default: DTLS port (20220).
+#DTLS_PORT = 20220
```

Comment:
What about using `DTLS_PORT ?= 20220` (since this value is the default) and let the possibility for user to set this value with the build command: `DTLS_PORT=22221 make -C examples/dtls-echo` ?
Then there's no need to comment out this block